includes/CAPx/Drupal/Mapper/MapperAbstract.php

```diff
@@ -52,10 +51,93 @@ public function getRemoteDataByJsonPath($data, $path) {
       throw new \Exception("Path cannot be empty", 1);
     }
 
+    // Wildcard paths need to have index association maintained. JSONPath does
+    // not provide much help in this and we need to do funny things.
```

Comment:
Funny as in like: ![nervous laughter](https://crownite.files.wordpress.com/2016/04/635916970850635301-873389412_tumblr_static_nervous_laughter.gif)